### ISSUE TITLE
getUpdatedTrackables keeps returning after removing anchor

### ISSUE THREAD

**Issue by rkardo58** [google-ar/arcore-android-sdk#800]
### SPECIFIC ISSUE ENCOUNTERED
Hi,

everytime i remove a anchor from my scene it seems that the getUpdatedTrackables takes ages to update what means that on my onUpdate will keep adding new models making it look like i never remove it.

Heres my onUpdate code:
`@Override
    public void onUpdate(FrameTime frameTime) {

        Frame frame = arFragment.getArSceneView().getArFrame();
        Collection<AugmentedImage> images = frame.getUpdatedTrackables(AugmentedImage.class);
        if (oldImages == null || images != oldImages){
            for (AugmentedImage image : images){
                if (image.getTrackingState().equals(TrackingState.TRACKING)){
                    oldImages = images;
                    if (image.getName().equals("fox.jpg") && !modelsIn.contains("fox")){
                        Anchor anchor = image.createAnchor(image.getCenterPose());
                        addingModel = true;
                        createModel(anchor, "fox");
                        modelsIn.add("fox");

                    } else if (image.getName().equals("ghost.jpg") && !modelsIn.contains("ghost")){
                        Anchor anchor = image.createAnchor(image.getCenterPose());
                        addingModel = true;
                        createModel(anchor, "ghost");
                        modelsIn.add("ghost");
                    }
                }
            }
        }
    }`

and here is my code to remove models
`private void removeAnchorNode(AnchorNode nodeToremove) {
        //Remove an anchor node
        if (nodeToremove != null) {
            arFragment.getArSceneView().getScene().removeChild(nodeToremove);
            nodeToremove.getAnchor().detach();
            nodeToremove.setParent(null);
            nodeToremove = null;
            Toast.makeText(this, "Test Delete - anchorNode removed", Toast.LENGTH_SHORT).show();
            modelsIn.remove(0);
        } else {
            Toast.makeText(this, "Test Delete - markAnchorNode was null", Toast.LENGTH_SHORT).show();
        }
    }`

### VERSIONS USED
- Android Studio: 3.4.1
- ARCore SDK for Android: 1.9.0
- Device manufacturer, model, and O/S: Xiaomi Pocophone F1

**Comment by devbridie** [comment]
As a part of ongoing efforts to maintain ARCore repositories, I’m closing this issue as stale. If this is still an issue, you can use the newly [open-sourced Sceneform SDK sources](https://github.com/google-ar/sceneform-android-sdk/tree/master/sceneformsrc/sceneform/src/main/java/com/google/ar/sceneform) to try to diagnose the problem, or find a community-reared fork in which the issue has been addressed.
